src/Components/Navigation/Navigation.js

```diff
@@ -0,0 +1,40 @@
+import { NavLink } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const Navigation = ({ isMobile }) => {
+  const activeStyle = {
+    fontWeight: "bold",
```

Comment:
I don't think I have seen this way of doing classes, very cool, will this be an opportunity for a dark mode?